src/app/dynamic-forms/fields/cinchy-form-link.directive.ts

```diff
@@ -245,6 +245,8 @@ export class LinkDirective implements OnInit {
         dropdownDataset = this.getSortedList(dropdownDataset);
         dataSet.dropdownDataset = dropdownDataset;
         this.dropdownSetOptions = dropdownDataset ? dropdownDataset.options : [];
+        this.dropdownSetOptions = this.dropdownSetOptions.sort((a, b) => a.label.toLowerCase().localeCompare(b.label.toLowerCase()));
```

Comment:
Doesn't line 245 already call this.getSortedList() ? If that function isn't sorting it correctly, can you fix that instead of sorting it again on this line?